app/main.py

```diff
@@ -23,14 +23,16 @@ def generate_badges():
 
 
 def empty_directory():
-    for file in os.listdir(BADGES_FOLDER):
-        file_path = os.path.join(BADGES_FOLDER, file)
-        try:
-            if os.path.isfile(file_path):
-                os.unlink(file_path)
-            elif os.path.isdir(file_path): shutil.rmtree(file_path)
-        except Exception:
-            traceback.print_exc()
+	if not os.path.exists(os.path.join('static','badges')):
```

Comment:
Is `os.path.join('static','badges')` equal to `BADGES_FOLDER` ?
If yes, you should use the variable.

Reply:
sure!
Let me change it and add the doc-string as well! 
:D